packages/amplify-category-storage/provider-utils/awscloudformation/service-walkthroughs/s3-walkthrough.js

```diff
@@ -287,13 +287,13 @@ function checkIfAuthExists(context) {
   return authExists;
 }
 
-function migrate(projectPath, resourceName) {
+function migrate(context, projectPath, resourceName) {
   const resourceDirPath = path.join(projectPath, 'amplify', 'backend', category, resourceName);
 
   // Change CFN file
 
   const cfnFilePath = path.join(resourceDirPath, 's3-cloudformation-template.json');
-  const oldCfn = JSON.parse(fs.readFileSync(cfnFilePath, 'utf8'));
+  const oldCfn = context.amplify.readJsonFile(cfnFilePath, 'utf8');
```

Comment:
The deleted comment is here, you don't need to use encoding if the encoding is 'utf8'.